main.go

```diff
@@ -119,6 +125,36 @@ func run(p params) error {
 //go:embed htmlOutput.html.tpl
 var htmlOutputTemplate string
 
+func (j junit2jira) convertToSlack(tc []testCase) error {
+	if j.slackOutput == "" {
+		return nil
+	}
+	slackMsg := convertJunitToSlack(tc...)
+	if slackMsg == nil {
+		log.Printf("warning: no slack message set")
```

Comment:
As a user I wouldn't know what this means. But do we really need to warn about apparently successful cases?

Reply:
good point this left from the original code